### PR TITLE
[INLONG-11567][Manager] Optimize the original DAG of Airflow

### LINKED ISSUE #11567: [Improve][Manager] Optimize the original DAG of Airflow
### Description

Optimize the original DAGs of Airflow.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Zkplo**
Fixes #11567

### Motivation

Optimize the original DAGs of Airflow.

### Verifying this change

*(Please pick either of the following options)*

- [x] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### CODE REVIEW

File: inlong-manager/manager-schedule/src/test/resources/airflow/dag_creator.py [hunk 1]
```diff
@@ -98,7 +96,7 @@ def taskFunction(**context):
     print("Request Body: ", data)
     response = requests.post(url, params=params, headers=headers, json=data)
     if response.status_code == 200:
-        print(response.json())
+        print(response.json())  # Assuming to return JSON data
```
Comment (fuweng11) [inline inlong-manager/manager-schedule/src/test/resources/airflow/dag_creator.py:99]: Please print more detailed information.

File: inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py [hunk 2]
```diff
@@ -29,33 +29,46 @@
...
+    current_time = utc_time.strftime("%Y-%m-%d %H:%M:%S.%f")
     logging.info(f"Current time: {current_time}")
-    for dag_file in os.listdir(DAG_PATH):
-        if dag_file.endswith(".py") and dag_file.startswith("inlong_offline_task_"):
-            with open(DAG_PATH + dag_file, "r") as file:
-                line = file.readline()
-                while line and "end_offset_datetime_str" not in line:
+
+    conf = context.get('dag_run').conf
+    logging.info(f"conf: {conf}")
```
Comment (fuweng11) [inline inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py:43]: Ditto.

File: inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py [hunk 3]
```diff
@@ -29,33 +29,46 @@
...
+DAG_PREFIX = 'inlong_offline_task_'
 
 def clean_expired_dags(**context):
+
     original_time = context.get('execution_date')
     target_timezone = pytz.timezone("Asia/Shanghai")
     utc_time = original_time.astimezone(target_timezone)
-    current_time = utc_time.strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3] + "Z"
+    current_time = utc_time.strftime("%Y-%m-%d %H:%M:%S.%f")
     logging.info(f"Current time: {current_time}")
```
Comment (fuweng11) [inline inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py:40]: Ditto.

File: inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py [hunk 4]
```diff
@@ -29,33 +29,46 @@
...
-                    end_date_str = line.split("=")[1].strip().strip("\"")
-                logging.info(f"DAG end time: {end_date_str}")
-                if end_date_str:
+                    while line and "end_offset_datetime_str" not in line:
+                        line = file.readline()
+                    end_date_str = None
+                    row = line.split("=")
+                    if len(row) > 1:
+                        end_date_str = datetime.fromtimestamp(int(row[1].strip().strip("\"")) / 1000, tz=target_timezone)
+                    logging.info(f"DAG end time: {end_date_str}")
```
Comment (fuweng11) [inline inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py:58]: Ditto.

File: inlong-manager/manager-schedule/src/test/resources/airflow/dag_creator.py (hunk 1 above)
Comment (Zkplo) [reply inlong-manager/manager-schedule/src/test/resources/airflow/dag_creator.py]: sure. done.

File: inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py (hunk 2 above)
Comment (Zkplo) [reply inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py]: done.

File: inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py [hunk 5]
```diff
@@ -29,33 +29,46 @@
...
     utc_time = original_time.astimezone(target_timezone)
-    current_time = utc_time.strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3] + "Z"
-    logging.info(f"Current time: {current_time}")
-    for dag_file in os.listdir(DAG_PATH):
-        if dag_file.endswith(".py") and dag_file.startswith("inlong_offline_task_"):
-            with open(DAG_PATH + dag_file, "r") as file:
-                line = file.readline()
-                while line and "end_offset_datetime_str" not in line:
+    current_time = utc_time.strftime("%Y-%m-%d %H:%M:%S.%f")
+    logging.info(f"The execution time of this cleaning task is: {current_time}")
```
Comment (fuweng11) [inline inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py:40]:
```suggestion
    logging.info(f"The execution time of this cleaning task is: {current_time} for groupId={groupId}")
```

File: inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py [hunk 6]
```diff
@@ -29,33 +29,46 @@
...
-                    end_date_str = line.split("=")[1].strip().strip("\"")
-                logging.info(f"DAG end time: {end_date_str}")
-                if end_date_str:
+                    while line and "end_offset_datetime_str" not in line:
+                        line = file.readline()
+                    end_date_str = None
+                    row = line.split("=")
+                    if len(row) > 1:
+                        end_date_str = datetime.fromtimestamp(int(row[1].strip().strip("\"")) / 1000, tz=target_timezone)
+                    logging.info(f"The end time of '{dag_file}' is: {end_date_str}")
```
Comment (fuweng11) [inline inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py:58]:
```suggestion
                    logging.info(f"The end time of '{dag_file}' is: {end_date_str} for groupId={groupId}")
```

File: inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py [hunk 7]
```diff
@@ -29,33 +29,46 @@
...
                     try:
-                        if str(current_time) > str(end_date_str):
-                            dag_file_path = os.path.join(DAG_PATH, dag_file)
+                        if end_date_str and str(current_time) > str(end_date_str):
                             os.remove(dag_file_path)
-                            # Optionally, delete the end_date variable
                             logging.info(f"Deleted expired DAG: {dag_file}")
                     except ValueError:
-                        logging.error(f"Invalid date format for DAG {dag_file}: {end_date_str}")
+                        logging.error(f"Failed to delete {dag_file}: {end_date_str}")
```
Comment (fuweng11) [inline inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py:64]: Ditto.

File: inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py (hunk 5 above)
Comment (Zkplo) [reply inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py]: done.

File: inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py (hunk 6 above)
Comment (Zkplo) [reply inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py]: done.

File: inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py [hunk 8]
```diff
@@ -29,33 +29,47 @@
...
-            with open(DAG_PATH + dag_file, "r") as file:
-                line = file.readline()
-                while line and "end_offset_datetime_str" not in line:
+    current_time = utc_time.strftime("%Y-%m-%d %H:%M:%S.%f")
+
+    conf = context.get('dag_run').conf
+    groupId = conf.get('inlong_group_id')
+
+    logging.info(f"Execution parameters for this cleaning task: {conf}")
+    logging.info(f"The execution time of this cleaning task is: {current_time} for groupId = {groupId}")
```
Comment (fuweng11) [inline inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py:45]:
```suggestion
    logging.info(f"Execution parameters = {conf} for groupId = {groupId} and  execution time = {current_time}")
```

Comment (Zkplo) [reply inlong-manager/manager-schedule/src/test/resources/airflow/dag_cleaner.py]: done.

**Review by fuweng11 — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by vernedeng — APPROVED** [review]

**Pull request merged**